src/full_client.toit

```diff
@@ -325,11 +325,48 @@ interface PersistenceStore:
   */
   do [block] -> none
 
+class PersistentPacket_:
+  topic /string
+  payload /ByteArray
+  retain /bool
+
+  constructor .topic .payload --.retain:
+
 /**
 A persistence store that stores the packets in memory.
 */
 class MemoryPersistenceStore implements PersistenceStore:
-  UNIMPLEMENTED ::= "UNIMPLEMENTED"
+  storage_ /Map := {:}
+  id_ /int := 0
+
+  store topic/string payload/ByteArray --retain/bool -> int:
+    id := id_++
+    storage_[id] = PersistentPacket_ topic payload --retain=retain
+    return id
+
+  get persistent_id/int [block] [--if_absent] -> none:
```

Comment:
This interface is a little bit weird. Also the store/get feels like an odd pair naming-wise.

Reply:
Changed it, by exposing the PersistedPacket